src/main/resources/static/js/maps.js

```diff
@@ -0,0 +1,49 @@
+
+	var map; 	
+	var infoWindow; 	
+	var geocoder = new google.maps.Geocoder();
+	var messageLoc;
+	function mapStart(localisation)  
+	{  
+		var pos = new google.maps.LatLng(53.429805, 14.537883);
+		var mapOptions = {
+			zoom: 10,
+			center: pos,
+			mapTypeId: google.maps.MapTypeId.ROADMAP,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["google" is not defined.](https://www.codacy.com/app/matikz1110/PetFinder/file/2664859802/issues/source?bid=3239464&fileBranchId=3331330#l12)